Visual Studio Code (1.90.1, undefined, desktop)
Jupyter Extension Version: 2024.5.0.
Python Extension Version: 2024.8.1.
Pylance Extension Version: 2024.6.1.
Platform: linux (x64).
No workspace folder opened.
22:04:38.689 [warn] Exception while attempting zmq : libstdc++.so.6: cannot open shared object file: No such file or directory
22:04:38.690 [warn] Exception while attempting zmq (fallback) : libstdc++.so.6: cannot open shared object file: No such file or directory
22:04:38.813 [info] Attempting to start a server because of preload conditions ...
22:04:39.412 [info] Attempting to start a server because of preload conditions ...
22:04:40.112 [warn] Failed to get activated env vars for ~/.guix-home/profile/bin/python3 in 560ms
22:04:40.121 [info] Process Execution: ~/.guix-home/profile/bin/python3 -c "import site;print("USER_BASE_VALUE");print(site.USER_BASE);print("USER_BASE_VALUE");"
22:04:40.220 [info] Process Execution: ~/.guix-home/profile/bin/python3 -c "import jupyter;print('6af208d0-cb9c-427f-b937-ff563e17efdf')"
22:04:40.227 [info] Process Execution: ~/.guix-home/profile/bin/python3 -c "import notebook;print('6af208d0-cb9c-427f-b937-ff563e17efdf')"
22:04:40.511 [warn] Failed to get activated env vars for ~/.guix-home/profile/bin/python3 in 115ms
22:04:40.521 [info] Process Execution: ~/.guix-home/profile/bin/python3 -m jupyter kernelspec --version
22:04:41.002 [info] Starting Jupyter Server
22:04:41.047 [info] Process Execution: ~/.guix-home/profile/bin/python3 -m jupyter notebook --no-browser --notebook-dir="/home/~/git/~ht.com/notebooks/compose2kubernetes" --config=/tmp/5e9f816e-5361-4540-94ca-0bc0b2a283bb/jupyter_notebook_config.py --NotebookApp.iopub_data_rate_limit=10000000000.0
22:04:41.939 [info] Process Execution: ~/.guix-home/profile/bin/python3 /home/~/.vscode/extensions/ms-toolsai.jupyter-2024.5.0-linux-x64/pythonFiles/vscode_datascience_helpers/getServerInfo.py
22:04:41.951 [info] Process Execution: ~/.guix-home/profile/bin/python3 /home/~/.vscode/extensions/ms-toolsai.jupyter-2024.5.0-linux-x64/pythonFiles/vscode_datascience_helpers/getServerInfo.py
22:04:56.943 [info] Starting Kernel (Python 3 (ipykernel)) for '~/git/~ht.com/notebooks/compose2kubernetes/notebook.ipynb' (disableUI=false)
22:04:57.025 [info] http://localhost:8888/: Kernel started: dcf7d8b9-379f-4b20-b922-2f12f2378c11
22:04:57.726 [info] Started session for kernel startUsingRemoteKernelSpec:3853f239a29b6f054d9cb77338442605ef8708bd3ea56ff4400e163d845cda2f..python3./gnu/store/1w5v338qk5m8khcazwclprs3znqp6f7f-python-3.10.7/bin/python3././gnu/store/1w5v338qk5m8khcazwclprs3znqp6f7f-python-3.10.7/bin/python3#-m#ipykernel_launcher#-f#{connection_file}
22:04:57.730 [warn] Kernel completions not enabled for 'Python 3 (ipykernel)'. 
To enable Kernel completion for this language please add the following setting 
jupyter.completionTriggerCharacters = {/gnu/store/1w5v338qk5m8khcazwclprs3znqp6f7f-python-3.10.7/bin/python3: [<List of characters that will trigger completions>]}. 
For more information please see https://aka.ms/vscodeJupyterCompletion
22:04:57.732 [info] Kernel successfully started
22:19:26.294 [info] Dispose Kernel '~/git/~ht.com/notebooks/compose2kubernetes/notebook.ipynb' associated with '~/git/~ht.com/notebooks/compose2kubernetes/notebook.ipynb'
